scripts/index.js

```diff
@@ -0,0 +1,78 @@
+let tempValue = 70;
+
+const updateTempColor = (currentTemp) => {
+    const tempValueBox = document.getElementById("tempValue");
+    let color = "teal";
+    if (currentTemp >= 80) {
+        color = "red";
+    } else if (currentTemp >= 70) {
+        color = "orange";
+    } else if (currentTemp >= 60) {
+        color = "yellow";
+    } else if (currentTemp >= 50) {
+        color = "green";
+    }
+    tempValueBox.classList = color;
+}
+
+const updateGarden = (currentTemp) => {
+    const landscapeBox = document.getElementById("landscape");
+    let landscape = "🗻🐻‍❄️⛄️🌲⛄️🍂🌲🍁🌲🌲⛄️🐇🌲🗻🏔";
+    if (currentTemp >= 80) {
+        landscape = "🌵🏜_🐪_🐍🌵_🦂🏜_🐫🌵"
+    } else if (currentTemp >= 70) {
+        landscape = "🌸🦩🌴__🌷🌺🌿_🦜🌴_🌻🏝";
+    } else if (currentTemp >= 60) {
+        landscape = "🌾🍃_🍃_⛰_🍁⛰_🍂🌾🌾_🍃";
+    }
+    landscapeBox.textContent = landscape;
+}
+
+const updateTemp = tempValue => {
+    const tempValueBox = document.getElementById("tempValue");
+    tempValueBox.textContent = tempValue;
+    updateTempColor(tempValue);
+    updateGarden(tempValue)
+}
+
+const increaseTemp = () => {
+    tempValue += 1;
+    updateTemp(tempValue)
+};
+
+const decreaseTemp = () => {
+    tempValue -= 1;
+    updateTemp(tempValue)
+};
+
+const increaseTempButton = document.getElementById("increaseTempControl");
+increaseTempButton.addEventListener("click", increaseTemp);
+
+const decreaseTempButton = document.getElementById("decreaseTempControl");
+decreaseTempButton.addEventListener("click", decreaseTemp);
```

Comment:
This will work as written because the script tag is at the bottom of the page, but wrapping this section in a function and calling it using an DOMContentLoaded event is a safety measure that can help prevent potential issues.